scripts/variantstore/wdl/GvsFindHailRefuse.wdl

```diff
@@ -0,0 +1,132 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsFindHailRefuse {
```

Comment:
Super nit. I find this name confusing - can't not read it as the verb case "I refuse" rather than refuse as in trash. But I'm having trouble coming up with a better name.

Reply:
Yeah that's the problem... perhaps 'GvsFindHailCruft'?

Reply:
Cruft sounds good to me!